### PR TITLE
Update language-tour.md

### LINKED ISSUE #2227: Language Tour - mixins issue
Page URL: https://dart.dev/guides/language/language-tour#adding-features-to-a-class-mixins

Found a typo? 
To specify that only certain types can use the mixin — for example, `so your mixin can invoke` a method that it doesn’t define — use on to specify the required superclass:

Description of issue:
Maybe it should be `so your mixin can't invoke`?

### PR DESCRIPTION
**Pull request by bhumikasharma29**
Language tour - mixin issue resolved
Clearly defined the use of 'on' keyword using an appropriate example.
Fixes #2227

### CODE REVIEW

**Comment by kwalrath** [comment]
Changing the code broke the build because we auto-include code. Do we really need the new code? If so, we'll need to [update the code samples](https://github.com/dart-lang/site-shared/blob/master/doc/code-excerpts.md).

But first we should make sure the words are as clear as possible (beyond stylistic edits that I'll be happy to make, such as [avoiding we](https://developers.google.com/style/person)). Maybe @ahan (who filed the original issue) can take a look?

**Comment by bhumikasharma29** [comment]
I think adding the code sample makes the concept clearer. If you want me to make any changes please do tell.

File: src/_guides/language/language-tour.md [hunk 1]
```diff
@@ -3426,16 +3426,24 @@ mixin Musical {
...
-To specify that only certain types can use the mixin — for example,
-so your mixin can invoke a method that it doesn't define —
-use `on` to specify the required superclass:
+To restrict mixin's use to only classes which either extends or implements the class on which it is declared on - 
+we use the 'on' keyword.
+For example:
 
 <?code-excerpt "misc/lib/language_tour/classes/orchestra.dart (mixin-on)"?>
 ```dart
+class Musician{
```
Comment (kwalrath) [inline src/_guides/language/language-tour.md:3435]:
```suggestion
class Musician {
```

File: src/_guides/language/language-tour.md [hunk 2]
```diff
@@ -3426,16 +3426,24 @@ mixin Musical {
...
+  //...
+}
 mixin MusicalPerformer on Musician {
   // ···
 }
+class MusicBand extends Musician with MusicalPerformer{
```
Comment (kwalrath) [inline src/_guides/language/language-tour.md:3441]:
```suggestion
class MusicBand extends Musician with MusicalPerformer {
```

**Review by kwalrath — COMMENTED** [review]
I've tweaked the text and code a bit. Next I'll update the code source files.

**Comment by kwalrath** [comment]
@bhumikasharma29 please check out my changes. You can see them rendered here:

https://dartlang-org-staging-1.firebaseapp.com/guides/language/language-tour#adding-features-to-a-class-mixins

**Comment by kwalrath** [comment]
@bhumikasharma29 can you please take a look?

**Review by kwalrath — APPROVED** [review]

**Pull request merged**